### PR TITLE
Bugfix(stomp)  Fix issues processing binary message when that message be...

### PR DESCRIPTION
**Pull request by treyhyde**
...gins with chr(10)

Protobuf binary will often start messages with the newline character
 because that's the way the message gets packed into binary.

Don't think that the client wants you to trim() all the bodies.  Let's
leave the message intact.

Issues: APP-8498

### CODE REVIEW

File: src/CentralDesktop/Stomp/Connection.php [hunk 1]
```diff
@@ -647,7 +647,7 @@ function readFrame() {
                 $command = $v;
             }
         }
-        $frame = new Frame($command, $headers, trim($body));
+        $frame = new Frame($command, $headers, $body);
```
Comment (treyhyde) [inline src/CentralDesktop/Stomp/Connection.php]: this is the real fix

File: src/CentralDesktop/Stomp/Connection.php [hunk 2]
```diff
@@ -627,7 +627,7 @@ function readFrame() {
                 $end  = true;
                 $data = $this->_extractNextMessage();
             }
-            $len = strlen($data);
+            $len = mb_strlen($data, '8bit');
```
Comment (treyhyde) [inline src/CentralDesktop/Stomp/Connection.php]: correctness

**Comment by treyhyde** [comment]
@dstevenson

**Comment by treyhyde** [comment]
@osbornk

**Comment by treyhyde** [comment]
Fixes #5 as well

**Comment by osbornk** [comment]
Approve.

**Pull request merged**